SU2_CFD/include/solvers/CNEMOEulerSolver.hpp

```diff
@@ -0,0 +1,912 @@
+ * \version 6.1.0
+ */
+class CNEMOEulerSolver : public CSolver {
+protected:
+
+  unsigned short
+  nSpecies;	           /*!< \brief Number of species in the gas mixture. */
+
+  su2double
+  Gamma,               /*!< \brief Mixture Cp/Cv. */
+  Gamma_Minus_One;     /*!< \brief Mixture Cp/Cv - 1. */
+
+  su2double
+  Mach_Inf,            /*!< \brief Free stream Mach number. */
+  *Density,            /*!< \brief Free stream species density. */
+  Density_Inf,         /*!< \brief Free stream sdensity. */
+  Energy_ve_Inf,       /*!< \brief Vib.-el. free stream energy. */
+  Pressure_Inf,	       /*!< \brief Free stream pressure. */
+  *Velocity_Inf,	   /*!< \brief Free stream flow velocity. */
+  *MassFrac_Inf,       /*!< \brief Free stream species mass fraction. */
+  Temperature_Inf,     /*!< \brief Trans.-rot. free stream temperature. */
+  Temperature_ve_Inf;  /*!< \brief Vib.-el. free stream temperature. */
+
+  su2double
+  *lowerlimit,         /*!< \brief contains lower limits for conserved variables. */
+  *upperlimit;         /*!< \brief contains upper limits for conserved variables. */
+
+  su2double
+  *CD_Inv,           /*!< \brief Drag coefficient (inviscid contribution) for each boundary. */
+  *CL_Inv,           /*!< \brief Lift coefficient (inviscid contribution) for each boundary. */
+  *CSF_Inv,          /*!< \brief Sideforce coefficient (inviscid contribution) for each boundary. */
+  *CMx_Inv,          /*!< \brief x Moment coefficient (inviscid contribution) for each boundary. */
+  *CMy_Inv,          /*!< \brief y Moment coefficient (inviscid contribution) for each boundary. */
+  *CMz_Inv,          /*!< \brief z Moment coefficient (inviscid contribution) for each boundary. */
+  *CoPx_Inv,         /*!< \brief x Moment coefficient (inviscid contribution) for each boundary. */
+  *CoPy_Inv,         /*!< \brief y Moment coefficient (inviscid contribution) for each boundary. */
+  *CoPz_Inv,         /*!< \brief z Moment coefficient (inviscid contribution) for each boundary. */
+  *CFx_Inv,          /*!< \brief x Force coefficient (inviscid contribution) for each boundary. */
+  *CFy_Inv,          /*!< \brief y Force coefficient (inviscid contribution) for each boundary. */
+  *CFz_Inv,          /*!< \brief z Force coefficient (inviscid contribution) for each boundary. */
+  *Surface_CL_Inv,   /*!< \brief Lift coefficient (inviscid contribution) for each monitoring surface. */
+  *Surface_CD_Inv,   /*!< \brief Drag coefficient (inviscid contribution) for each monitoring surface. */
+  *Surface_CSF_Inv,  /*!< \brief Side-force coefficient (inviscid contribution) for each monitoring surface. */
+  *Surface_CEff_Inv, /*!< \brief Side-force coefficient (inviscid contribution) for each monitoring surface. */
+  *Surface_CFx_Inv,  /*!< \brief x Force coefficient (inviscid contribution) for each monitoring surface. */
+  *Surface_CFy_Inv,  /*!< \brief y Force coefficient (inviscid contribution) for each monitoring surface. */
```

Comment:
Would it make sense to inherit from CEulerSolver to re-use these things and possible their computation? (Some mad lad (me) spent more than a couple hours cleaning this code in CEulerSolver)